.github/workflows/compatibility.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         pack_kind: [current, previous]
         create_builder_kind: [current, previous]
-        lifecycle_kind: [current, previous]
+        lifecycle_kind: [default, previous]
```

Comment:
The underlying implementation of what `default` means here is the same as what `current` is in the existing tests. I wasn't aware that there's actually a distinction. I believed that our intent was to not test the actual `current` of lifecycle because that could break and cause PRs to be blocked for reasons outside of contributor's control.